patches/api/0014-Version-Command-2.0.patch

```diff
@@ -158,18 +182,18 @@ index 04b4fb6859df0221f8f9f92c5a7ac2dda1073355..b50f614806f4634960d383e8a33f094c
  
 -    private void setVersionMessage(@NotNull String msg) {
 +    // Paper start
-+    private void setVersionMessage(final @NotNull net.kyori.adventure.text.Component msg) {
++    private void setVersionMessage(final @NotNull Component msg) {
          lastCheck = System.currentTimeMillis();
 -        versionMessage = msg;
-+        final net.kyori.adventure.text.Component message = net.kyori.adventure.text.TextComponent.ofChildren(
-+            net.kyori.adventure.text.Component.text("This server is running " + Bukkit.getName() + " version " + Bukkit.getVersion() + " (Implementing API version " + Bukkit.getBukkitVersion() + ")", net.kyori.adventure.text.format.NamedTextColor.WHITE),
-+            net.kyori.adventure.text.Component.newline(),
++        final Component message = TextComponent.ofChildren(
```

Comment:
ofChildren is deprecated for removal. This PR should probably just replace it with the alternative defined in the jd.